samples/fabric/fabric-cli/scripts/getAssetTransferViewAddress.js

```diff
@@ -9,7 +9,7 @@ function getECertBase64(networkId, userId) {
     return Buffer.from(userIdJSON.credentials.certificate).toString('base64');
```

Comment:
Do we need this script anymore?

Reply:
No, we don't, thought to keep it till my changes were finalized. Should I delete them?

Reply:
Yes, you can delete them. If we ever need to refer to them, we can look at the revision history.